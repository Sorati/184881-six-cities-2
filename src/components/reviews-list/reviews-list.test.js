import renderer from 'react-test-renderer';
import ReviewsList from '../reviews-list/reviews-list';

it(`Reviews list correctly renders with 2 reviews`, () => {

  const reviews = [
    {
      user: {
        photo: `/img/apartment-02.jpg`,
        name: `Max`
      },
      rating: 72,
      text: `Great place!`,
      date: `2019-07-12`
    }, {
      user: {
        photo: `/img/apartment-01.jpg`,
        name: `John`
      },
      rating: 23,
      text: `Not great place!`,
      date: `2019-07-10`
    }
  ];
  const tree = renderer
    .create(<ReviewsList reviews={reviews}/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
