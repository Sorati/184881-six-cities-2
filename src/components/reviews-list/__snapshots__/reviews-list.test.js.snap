// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Reviews list correctly renders with 2 reviews 1`] = `
<ul
  className="reviews__list"
>
  <li
    className="reviews__item"
  >
    <div
      className="reviews__user user"
    >
      <div
        className="reviews__avatar-wrapper user__avatar-wrapper"
      >
        <img
          alt="Reviews avatar"
          className="reviews__avatar user__avatar"
          height={54}
          src="/img/apartment-02.jpg"
          width={54}
        />
      </div>
      <span
        className="reviews__user-name"
      >
        Max
      </span>
    </div>
    <div
      className="reviews__info"
    >
      <div
        className="reviews__rating rating"
      >
        <div
          className="reviews__stars rating__stars"
        >
          <span
            style={
              Object {
                "width": "72%",
              }
            }
          />
          <span
            className="visually-hidden"
          >
            Rating
          </span>
        </div>
      </div>
      <p
        className="reviews__text"
      >
        Great place!
      </p>
      <time
        className="reviews__time"
        dateTime="2019-07-12"
      >
        July 2019
      </time>
    </div>
  </li>
  <li
    className="reviews__item"
  >
    <div
      className="reviews__user user"
    >
      <div
        className="reviews__avatar-wrapper user__avatar-wrapper"
      >
        <img
          alt="Reviews avatar"
          className="reviews__avatar user__avatar"
          height={54}
          src="/img/apartment-01.jpg"
          width={54}
        />
      </div>
      <span
        className="reviews__user-name"
      >
        John
      </span>
    </div>
    <div
      className="reviews__info"
    >
      <div
        className="reviews__rating rating"
      >
        <div
          className="reviews__stars rating__stars"
        >
          <span
            style={
              Object {
                "width": "23%",
              }
            }
          />
          <span
            className="visually-hidden"
          >
            Rating
          </span>
        </div>
      </div>
      <p
        className="reviews__text"
      >
        Not great place!
      </p>
      <time
        className="reviews__time"
        dateTime="2019-07-10"
      >
        July 2019
      </time>
    </div>
  </li>
</ul>
`;
